dashboard/test/mailers/previews/pd_workshop_mailer_preview.rb

```diff
@@ -252,6 +252,13 @@ def teacher_enrollment_reminder__csp_for_returning_teachers_3_day
       options: {days_before: 3}
   end
 
+  def teacher_virtual_order_form_reminder
+    mail :teacher_virtual_order_form_reminder, Pd::Workshop::COURSE_CSP, Pd::Workshop::SUBJECT_CSP_SUMMER_WORKSHOP,
```

Comment:
Is it ok that this references CSP but some of the above code looks like it can also be used with CSD?

Reply:
ahh good catch, yeah that should work for both

Reply:
Ok, maybe add a comment to indicate that this is intentional (or change the constant name so it's not specific to CSP)?

Reply:
updated name and added a second for CSD

Reply:
Looks great, thanks!